### ISSUE TITLE
[Checkbox] Labels for single checkbox are not announced by NVDA in browse mode (navigation with arrows) 

### ISSUE THREAD

**Issue by a11ymiko** [sl-design-system/components#3338]
Checkbox labels are not announced by NVDA in browse mode (navigation with arrows) for Checkbox With Tooltip story. Checkboxes are announced as 'clicable chechk box required invalid entry' without name, that is provided by label.
This is announced properly in Checkbox group With Tooltips story.

Current behavior:

https://github.com/user-attachments/assets/78bb0866-590a-4f72-b8c0-d36d2bfa14e3

Expected behavior:

https://github.com/user-attachments/assets/6693bda8-0a8b-4496-9406-1c174ce3d88b

_Originally posted by @a11ymiko in https://github.com/sl-design-system/components/pull/3322#pullrequestreview-4291320861_

**Comment by a11ymiko** [comment]
Current behavior on production Storybook is correct. Labels are announced in browse mode: 

https://github.com/user-attachments/assets/3048a684-0a20-438f-b384-67ca38c6b5d5

This is also true for https://github.com/sl-design-system/components/pull/3605 where support for tooltip is added for checkbox and radio.
